am/templates/api/api-deployment.yaml

```diff
@@ -105,6 +105,30 @@ spec:
             - name: GRAVITEE_PLUGINS_PATH_1
               value: '${gravitee.home}/plugins-ext'
             {{- end }}
+            {{- if .Values.cockpit.enabled }}
+            - name: gravitee_cockpit_enabled
+              value: "true"
+            - name: gravitee_cockpit_url
+              value: {{ .Values.cockpit.url }}
+            - name: gravitee_cockpit_ws_endpoints_0
+              value: {{ .Values.cockpit.controller }}
+            - name: gravitee_cockpit_keystore_type
+              value: pkcs12
+            - name: gravitee_cockpit_keystore_path
+              value: /opt/graviteeio-management-api/cockpit/keystore.p12
```

Comment:
according to L169, the directory should be `graviteeio-am-management-api`. wdyt  ?